src/compare.ts

```diff
@@ -65,9 +79,14 @@ async function diffPair(
         const { width, height } = baseline;
         const diff = new PNG({ width, height });
 
-        const numDiffPixels = pixelmatch(baseline.data, test.data, diff.data, width, height, {
-            threshold: 0.1,
```

Comment:
Default threshold was 0.1, now it's undefined and our tests break. It will effect android VRT runner too. Please be careful about this.

Reply:
Returned it